camera360_insta: {
  front: {
    sub_topics: [
      "/camera360/front_lens/front_projection/image_rect/compressed",
    ],
    pub_topics: "front",
    stitch_calib_file: "",
    bev_calib_file: "$(find camera360_perception)/cfg/bev/bev_front.json",
    camera_intrinsic_file_path: "$(find camera360_perception)/cfg/cameras_info/insta360_air/camera_intrinsic.yaml",
    base_frame: "camera360_center_link",
    pose_to_base_frame: [0,0,0,0,0,0],
    occ_grid_size: [20,30]
  },

  rear: {
    sub_topics: [
      "/camera360/rear_lens/front_projection/image_rect/compressed"
    ],
    pub_topics: 'rear',
    stitch_calib_file: "",
    bev_calib_file: "/$(find camera360_perception)/cfg/bev/bev_rear.json",
    camera_intrinsic_file_path: "$(find camera360_perception)/cfg/cameras_info/insta360_air/camera_intrinsic.yaml",

    base_frame: "camera360_center_link",
    pose_to_base_frame: [0,0,0,0,0,3.1415],
    occ_grid_size: [20,30]

  },

  left: {
    # stitching requires input ordered from leftmost image fragment to rightmost image fragment
    sub_topics: [
      "/camera360/rear_lens/right_projection/image_rect/compressed",
      "/camera360/front_lens/left_projection/image_rect/compressed",

    ],
    pub_topics: "left",
    stitch_calib_file: "$(find camera360_perception)/cfg/stitch/stitch_left.yaml",
    bev_calib_file: "$(find camera360_perception)/cfg/bev/bev_left.json",
    camera_intrinsic_file_path: "$(find camera360_perception)/cfg/cameras_info/insta360_air/camera_intrinsic_right.yaml",

    base_frame: "camera360_center_link",
    pose_to_base_frame: [0,0,0,0,0,1.57],
    occ_grid_size: [20,10]

  },

  right: {
   sub_topics: [
      "/camera360/front_lens/right_projection/image_rect/compressed",
      "/camera360/rear_lens/left_projection/image_rect/compressed",
    ],
    pub_topics: "right",
    stitch_calib_file: "$(find camera360_perception)/cfg/stitch/stitch_right.yaml",
    bev_calib_file: "$(find camera360_perception)/cfg/bev/bev_right.json",
    camera_intrinsic_file_path: "$(find camera360_perception)/cfg/cameras_info/insta360_air/camera_intrinsic_right.yaml",
    base_frame: "camera360_center_link",
    pose_to_base_frame: [0,0,0,0,0,-1.57],
    occ_grid_size: [20,10]
  },
}
model_path: "$(find camera360_perception)/scripts/pretrained_models/enet.pb"
# camera_intrinsic_file_path: "$(find camera360_perception)/cfg/camera_intrinsic.yaml"